whedon_api.rb

```diff
@@ -301,12 +301,12 @@ def deposit(dry_run)
       end
 
       if dry_run == true
+        label_issue(@nwo, @issue_id, ['recommended-accepted'])
         respond "```\nAttempting dry run of processing paper acceptance...\n```"
         DOIWorker.perform_async(@nwo, @issue_id, serialized_config, custom_branch=nil, clear_cache=false)
         DepositWorker.perform_async(@nwo, @issue_id, serialized_config, dry_run=true)
       else
-        label_issue(@nwo, @issue_id, ['accepted'])
-
+        label_issue(@nwo, @issue_id, ['accepted', 'published'])
```

Comment:
When an EiC accepts a paper, the labels `accepted` and `published` labels are added.